core/server/api/canary/utils/validators/utils/json-schema.js

```diff
@@ -5,7 +5,17 @@ const common = require('../../../../../lib/common');
 
 const ajv = new Ajv({
     allErrors: true,
-    useDefaults: true
+    useDefaults: true,
+    formats: {
+        'json-string': (data) => {
+            try {
+                JSON.parse(data);
```

Comment:
@daniellockyer was having a quick look at this and am wondering about speed implications having to parse quite large json on every request. If the goal is to validate our mobiledoc fields maybe having a very simple regex that checks for mobiledoc format would be more appropriate solution?  :thinking: